### DISCUSSION TITLE
I HAVE A QUESTION REGARDING SPOTIFLYER

### DISCUSSION THREAD

**Discussion by reydelshit** [Shabinder/SpotiFlyer#1598] [General]
Hello good evening, developers!

I am an aspiring web developer, I am trying to create a Spotify music downloader which will be based on web. I am asking how did guys do it on spotiflyer?

Thank you !

**Comment by MrAn0nym** [comment]
> I am asking how did guys do it on spotiflyer?

https://github.com/Shabinder/SpotiFlyer/tree/main/web-app xD

**Reply by reydelshit** [reply]
I'm trying to create a project which is similar to that (if already existed). So, I am asking how did you guys do it? or just tell me the specific term so that i can search it and study
